plugins/modules/monitoring/spectrum_model_attrs.py

```diff
@@ -0,0 +1,526 @@
+            "isManaged": "true"
+        },
+    }
+'''
+
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils._text import to_native
+from ansible.module_utils.urls import fetch_url
+import json
+import re
+import xml.etree.ElementTree as ET
+try:
+    # python3
+    import urllib.parse as urlparse
+except ImportError:
+    # python2
+    import urlparse
+
+
+class spectrum_model_attrs:
+    def __init__(self, module):
+        self.module = module
+        self.url = module.params['url']
+        # If the user did not define a full path to the restul space in url:
+        # params, add what we believe it to be.
+        if not re.search('\\/.+', self.url.split('://')[1]):
+            self.url = "%s/spectrum/restful" % self.url.rstrip('/')
+        # Align these with what is defined in OneClick's UI under:
+        # Locator -> Devices -> By Model Name -> <enter any model> ->
+        #    Attributes tab.
+        self.attr_map = dict(App_Manufacturer=hex(0x230683),
+                             CollectionsModelNameString=hex(0x12adb),
+                             Condition=hex(0x1000a),
+                             Criticality=hex(0x1290c),
+                             DeviceType=hex(0x23000e),
+                             isManaged=hex(0x1295d),
+                             Model_Class=hex(0x11ee8),
+                             Model_Handle=hex(0x129fa),
+                             Model_Name=hex(0x1006e),
+                             Modeltype_Handle=hex(0x10001),
+                             Modeltype_Name=hex(0x10000),
+                             Network_Address=hex(0x12d7f),
+                             Notes=hex(0x11564),
+                             ServiceDesk_Asset_ID=hex(0x12db9),
+                             TopologyModelNameString=hex(0x129e7),
+                             sysDescr=hex(0x10052),
+                             sysName=hex(0x10b5b),
+                             Vendor_Name=hex(0x11570),
+                             Description=hex(0x230017))
+        self.search_qualifiers = [
+            "and", "or", "not", "greater-than", "greater-than-or-equals",
+            "less-than", "less-than-or-equals", "equals", "equals-ignore-case",
+            "does-not-equal", "does-not-equal-ignore-case", "has-prefix",
+            "does-not-have-prefix", "has-prefix-ignore-case",
+            "does-not-have-prefix-ignore-case", "has-substring",
+            "does-not-have-substring", "has-substring-ignore-case",
+            "does-not-have-substring-ignore-case", "has-suffix",
+            "does-not-have-suffix", "has-suffix-ignore-case",
+            "does-not-have-suffix-ignore-case", "has-pcre",
+            "has-pcre-ignore-case", "has-wildcard", "has-wildcard-ignore-case",
+            "is-derived-from", "not-is-derived-from"]
+
+        self.resp_namespace = \
+            dict(ca="http://www.ca.com/spectrum/restful/schema/response")
```

Comment:
```suggestion
        self.resp_namespace = dict(ca="http://www.ca.com/spectrum/restful/schema/response")
```
There is enough space in this line to avoid `\`.